mono/metadata/assembly.c

```diff
@@ -1911,13 +1911,13 @@ static AssemblyPreLoadHook *assembly_preload_hook = NULL;
 static AssemblyPreLoadHook *assembly_refonly_preload_hook = NULL;
 
 static MonoAssembly *
-invoke_assembly_preload_hook (MonoAssemblyName *aname, gchar **assemblies_path)
```

Comment:
Curiousity: why the need of changing the name?

Reply:
Probably it masks a global. I think msvc maybe goes overboard in what it thinks is worth warning about. Lexical scoping is a deliberate feature, usually accepted as common sense but sometimes criticized.

Reply:
Several of the name changes fixes MSVC warnings where local variables hides global variables, using the same name.